website/pages/docs/commands/plugin/reload.mdx

```diff
@@ -33,3 +33,6 @@ flags](/docs/commands) included on all commands.
 
 - `-mounts` `(array: [])` - Array or comma-separated string mount paths of the
   plugin backends to reload.
+
+- `-scope` `(string: "")` - The scope of the reload, omitted for local, 'global'
```

Comment:
Can we re-phrase "omitted for local", as local could it mean mounts, namespaces, or nodes.

Reply:
Will do. I just copied what was in the CLI help text. I will adjust that in a later PR after re-wording this one.

Reply:
@calvn What are your thoughts on the new phrasing?